src/pass.h

```diff
@@ -69,14 +69,15 @@ struct InliningOptions {
   // More generally, with 2 items we may have a local.get, but no way to
   // require it to be saved instead of directly consumed.
   Index alwaysInlineMaxSize = 2;
-  // Function size which we inline when functions are lightweight (no loops
-  // and calls) and we are doing aggressive optimisation for speed (-O3).
-  // In particular it's nice that with this limit we can inline the clamp
-  // functions (i32s-div, f64-to-int, etc.), that can affect perf.
-  Index flexibleInlineMaxSize = 20;
   // Function size which we inline when there is only one caller.
   // FIXME: this should logically be higher than flexibleInlineMaxSize.
```

Comment:
should we address this FIXME and indeed make `oneCallerInlineMaxSize` > `flexibleInlineMaxSize` by default?

Reply:
I don't think so. I think we should allow it, but the current values make sense to me. But, we should do more measuring - I have more improvements here that I'll be testing.